evalml/automl/utils.py

```diff
@@ -63,7 +64,7 @@ def make_data_splitter(X, y, problem_type, problem_configuration=None, n_splits=
     if problem_type == ProblemTypes.REGRESSION:
```

Comment:
Hmmm, that's a good question. It wasn't something I was going to add in this iteration, but it might be good to add? It's untested in terms of perf tests, however. @dsherry what do you think?